src/app/clients/page.tsx

```diff
@@ -9,16 +9,24 @@ const dataFetch: () => Promise<Schema["clients"][] | null> = async () => {
 };
 
 const Clients: () => Promise<React.ReactElement> = async () => {
-    const data = await dataFetch();
+    let data: Datum[] | null = await dataFetch();
```

Comment:
potentially better as
`let data: Datum[] = await dataFetch() ?? []`

Reply:
(which then also absorbs the if statement below)

Reply:
Refactored as suggested